docs/deploy_terraform.md

```diff
@@ -17,7 +17,7 @@ sudo chmod +x /usr/local/bin/packer
 
 Install Terraform:
 ```bash
-sudo curl -L "https://releases.hashicorp.com/terraform/0.13.0-beta3/terraform_0.13.0-beta3_linux_amd64.zip" -o /usr/local/bin/terraform
+sudo curl -L "https://releases.hashicorp.com/terraform/0.13.4/terraform_0.13.4_linux_amd64.zip" -o /usr/local/bin/terraform
```

Comment:
Thanks Aniek. Does 0.13.5 work on Mac? I'm using that version on Linux now.

Reply:
Just checked 0.13.5 and works without issues :-). I noticed that the terraform cloud workspace versions now can also be updated.

If you could update the version to "0.13.5" here that would be great:
https://github.com/The-Academic-Observatory/observatory-platform/blob/2eaccf376805a3f7f59fce6b6bb9fb649bc66421/observatory-platform/observatory/platform/terraform_api.py#L152

I also made a separate organisation for unit testing a while back, but didn't use it for tests yet as it didn't have the version available that we needed for some functionalities. We could use that one now, or create another one with a different name if you'd prefer. (The one I made is called observatory-unit-testing). Just need to update the token in github secrets if we do.

Reply:
Thanks Aniek.

Yeah it would be good to use the observatory-unit-testing Terraform organisation for tests, I added the an extra Github secret that you can supply the organisation name with: `TESTS_TERRAFORM_ORGANISATION`